middlewares/authn.js

```diff
@@ -8,6 +8,9 @@ var sqlLoader = require('../prairielib/lib/sql-loader');
 
 var sql = sqlLoader.loadSqlEquiv(__filename);
 
+const UUID_REGEXP =
+  /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/;
```

Comment:
Nit: I don't think the `\b` entries are doing anything here, given that we've precisely specified which characters must come before and after the `\b`.

Reply:
Fair enough, I just copied this from a "uuid regexp" Google search result 😂

Reply:
Simplified in ca49a3a.